### PR TITLE
Replaces the fireball book in the Drake chest with a book of Sacred Flame

### PR DESCRIPTION
**Pull request by Ergovisavi**
:cl:
add: Adds "Sacred Flame", a spell that makes everyone around you more flammable, and lights yourself on fire. Share the wealth. Of fire.
tweak: Replaces the fireball spellbook as a possible reward from a Drake chest with a book of sacred flame.
/:cl:
- Adds "Sacred Flame", which makes everyone (including you) more flammable and ignites yourself.
- Replaces the fireball spellbook in one of the Dragon chests with a spellbook of Sacred Flame (But leaves the wand)

sacred_flame_book Touch the demon inside you

Making this change because I feel like fireball is not a super great mechanic (Throwable while stunned, RNG knockout, proxdet in 3x3 hallways) in non wizard rounds, and I honestly feel like a piece of shit whenever I hit people with it as a miner. I'm still leaving in the wand of of fireball, but that only has a few uses, but isn't usable while stunned, can be taken from you, etc.

Plus, sacred flame can't hurt you if you're wearing the armor you get from killing an ash drake. Should be plenty of hilarity in a crowded shuttle.

### CODE REVIEW

**Comment by yackemflam** [comment]
There's a book of charge in mining.
BoC + fireball wand > spell

**Comment by Ergovisavi** [comment]
There's a PR up for the removal of that.

**Comment by Kromgar** [comment]
Yeah the fireball toggle is sooo bad. You can't untoggle it unless it is charged.


But maybe you could give the wand a couple of extra charged?

**Comment by PKPenguin321** [comment]
i feel like lighting yourself on fire at will is kind of a stupid reward for killing a drake

you know that drake attack that shoots fire tiles in a + shape around him? Maybe make it do that but with way way way less range and a decent cooldown

File: code/modules/spells/spell_types/wizard.dm [hunk 1]
```diff
@@ -385,3 +385,23 @@
...
+	range = 6
+	include_user = 1
+	selection_type = "view"
+	action_icon_state = "sacredflame"
+	sound = "sound/magic/Fireball.ogg"
+
+/obj/effect/proc_holder/spell/targeted/sacred_flame/cast(list/targets, mob/user = usr)
+	for(var/mob/living/L in targets)
+		L.adjust_fire_stacks(20)
+		if(L == user)
```
Comment (RemieRichards) [inline code/modules/spells/spell_types/wizard.dm]: why do this? why not just after the loop do ```user.IgniteMob()``` (casting w/e if needbe)

**Comment by Kromgar** [comment]
Also you should make sacred flame kill weak blob tiles around the caster. Because fireballs were awesome for blob this wouldnt work on blob at all

**Comment by Ergovisavi** [comment]
@PKPenguin321 
>i feel like lighting yourself on fire at will is kind of a stupid reward for killing a drake
>you know that drake attack that shoots fire tiles in a + shape around him? Maybe make it do that but with way way way less range (maybe like 3 tiles on all sides) and a decent cooldown

Keep in mind that it lights you on fire (But you're immune to the effects due to wearing the drake armor of the drake you just killed), makes everyone hit the fire stack cap (Which means they either find an extinguisher or perish, rolling four times is likely going to still leave them in crit), and you can ignite them just by running into them. And then those people can bump into other people, and the process can repeat. 

Miners also have access to stimulants to make them run faster. Even if someone shoots you with a taser, if they accidentally touch you, they're on fire and need to deal with the situation. Yes, it's weaker than fireball, but _that's the point_. You're also still getting a wand of fireball, which is a very, very potent tool.

File: code/modules/spells/spell_types/wizard.dm (hunk 1 above)
Comment (Ergovisavi) [inline code/modules/spells/spell_types/wizard.dm]: I am not good with computer

I'll fix that when I get home.

**Comment by PKPenguin321** [comment]
i guess but my idea is still cooler

**Comment by Boredone** [comment]
Could be nice for wizards (Pretty sure their hardsuit is fire immune as well) so maybe you could toss it on the spell book as well?

**Comment by ChangelingRain** [comment]
Fireball is super OP in the hands of non-wizards honestly

**Comment by PKPenguin321** [comment]
u gave me merge conflicts :fist:

**Pull request merged**